lib/tests/testscripts/xml/testscript-search.xml

```diff
@@ -188,7 +190,7 @@
         <type>
           <code value="search"/>
         </type>
-        <resource value="Patient"/>
+        <resource value="Patient"/> <!-- should this return a bundle or patient? -->
```

Comment:
Yeah that's what I figured.  Their example test script originally made it so that a Patient was expected, but I figured that was obviously wrong so I updated the test script to look for a bundle, but forgot to remove this comment.  I'll remove this comment.